browser/p3a/p3a_core_metrics.cc

```diff
@@ -172,4 +176,78 @@ void BraveWindowTracker::UpdateP3AValues() const {
   UMA_HISTOGRAM_EXACT_LINEAR("Brave.Core.TorEverUsed", tor_used, 1);
 }
 
+void BraveNewTabCountTracker::CreateInstance(PrefService* local_state) {
+  g_brave_ntp_count_tracker_instance = new BraveNewTabCountTracker(local_state);
+}
+
+void BraveNewTabCountTracker::RegisterPrefs(PrefRegistrySimple* registry) {
+  registry->RegisterListPref(kNewTabsCreated);
+  registry->RegisterListPref(kSponsoredNewTabsCreated);
+}
+
+BraveNewTabCountTracker::BraveNewTabCountTracker(PrefService* local_state)
+    : new_tab_count_state_(local_state, kNewTabsCreated),
+      branded_new_tab_count_state_(local_state, kSponsoredNewTabsCreated) {
+  ::ntp_background_images::ViewCounterService::AddObserver(this);
+}
+
+BraveNewTabCountTracker::~BraveNewTabCountTracker() {
+  ::ntp_background_images::ViewCounterService::RemoveObserver(this);
```

Comment:
the tracker is leaking anyway, so there is no sense in having a destructor

Reply:
Should this similar call be removed from `BraveWindowTracker` also?

Reply:
yeah, it seems so

Reply:
will leave alone - I had backed out `BraveNewTabCountTracker` code 😄